backend/app/tests/api/routes/test_threads.py

```
@@ -27,7 +27,7 @@
 
 
 @patch("app.api.routes.threads.OpenAI")
-def test_threads_endpoint(mock_openai, db):
+def test_threads_endpoint(mock_openai, db, user_api_key_header):
```

Ruff (0.12.2)

Unused function argument: `db`

(ARG001)